src/views/Restoration.vue

```diff
@@ -95,13 +104,21 @@ import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
 import { BcRestorationResource, BenRestorationResource, CccRestorationResource, UlcRestorationResource }
   from '@/resources/Restoration/'
 import { FilingDataIF } from '@bcrs-shared-components/interfaces'
+import { RelationshipsPanel } from '@bcrs-shared-components/relationships-panel'
+import CourtOrderPoa from '@/components/common/CourtOrderPoa.vue'
+import { LimitedRestorationPanel } from '@bcrs-shared-components/limited-restoration-panel'
+import QuestionWrapper from '@/components/common/QuestionWrapper.vue'
 
 @Component({
   components: {
+    QuestionWrapper,
+    CourtOrderPoa,
     CertifySection,
     CurrentDirectors,
     DocumentsDelivery,
+    LimitedRestorationPanel,
```

Comment:
These components aren't needed yet but will be needed prior to completion of this form.